arches/app/datatypes/datatypes.py

```diff
@@ -744,7 +744,7 @@ def transform_value_for_tile(self, value, **kwargs):
                     value = datetime.strptime(value, valid_date_format)
                 else:
                     value = datetime.strptime(value, settings.DATE_IMPORT_EXPORT_FORMAT)
-            if isinstance(value, date):
+            elif type(value) is date:
```

Comment:
I think it would be good to clarify why you're using type instead of isinstance as is used above.
```suggestion
            elif type(value) is date: # Use type() instead of isinstance() to distinguish between date and datetime
```